### DISCUSSION TITLE
New user questions

### DISCUSSION THREAD

**Discussion by kingspawn1** [moneytoo/Player#273] [Q&A]
I'm new to Just Player (Legacy) and have been trying to find information that would normally be found in a user manual (I'm at the 'trying to learn how to navigate around the program to get to various settings and features). If someone could point me to that type of documentation it would be greatly appreciated.

Specifically right now I'm trying to understand how 'Volume Boost' works and how to access any settings for that feature that I can control.

Thanks in advance for your help.

**Comment by moneytoo** [comment]
The volume boost feature should increase the volume past the normal maximum (using [LoudnessEnhancer](https://developer.android.com/reference/android/media/audiofx/LoudnessEnhancer)). Normally, you should access this just by increasing the volume using either a touch screen gestures or volume up key presses.
Though this isn't really accessible on Android TV devices where the app doesn't receive volume key presses. I guess I should mention that in the [README.md](https://github.com/moneytoo/Player/blob/master/README.md).

**Reply by brentonv** [reply]
> this isn't really accessible on Android TV devices where the app doesn't receive volume key presses

It would be nice if it did

**Reply by kingspawn1** [reply]
I was hoping that i could go into a settings screen somewhere and tell just player to 'boost' the audio by some amount i indicate with like a slider control. Then i could tweak the volume from there using the streaming device volume controls (just player would not be involved at this point) on the remote while watching the video.

**Reply by moneytoo** [reply]
@brentonv I feel that this isn't as much needed on TVs as the speakers/AVRs are usually powerful enough. But I guess it could still be useful especially for BT headphones. Let me know if you find any seamless implementation of such volume boosting feature in other video player app.

**Reply by kingspawn1** [reply]
You are correct, I am feeding the audio from my Nvidia Shield Pro to BT devices (Apple Airpod Pros and my wife's hearing aids) and they have no built in volume adjustment. So I need to have the player boost the audio to a higher level by default and I then use the shields remote volume control to set the level going to the airpods/hearing aids. I have found that the Kodi player has what I need for volume control and it works fine for that but in other ways i'm not happy with kodi and prefer just player. In the Kodi audio settings there are two sliders, one for 'volume' and a second for 'volume amplification'. Between those to settings I can get it to work for my situation. One other thing about the settings in Kodi - I can leave the setting changes temporary (just for the current video) or I can save the settings as default for all future videos.

Thanks!

**Reply by jpants36** [reply]
> @brentonv I feel that this isn't as much needed on TVs as the speakers/AVRs are usually powerful enough. But I guess it could still be useful especially for BT headphones. Let me know if you find any seamless implementation of such volume boosting feature in other video player app.

MXPlayer has a volume booster on TV. You can use it by using the up and down keys while watching something.

**Reply by moneytoo** [reply]
> MXPlayer has a volume booster on TV. You can use it by using the up and down keys while watching something.

The thing is that up and down buttons are used to activate the ui when video is playing as this is what is expected from a video player app: https://developer.android.com/training/tv/playback/controls
